doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -403,60 +417,209 @@ Response code:
             }
         }
 
-### Retrieve entity data by Id. Not exists [GET /v2/entities/{entityId}]
+### Update or append entity attributes [POST /v2/entities/{entityId}?{op}]
+
+<table style="width: 100%; margin: 12px 0 0 0;">
+<tr><td colspan="2"><strong>Required Parameters</strong></td></tr>
+<tr><td colspan="2">none</td></tr>
+<tr><td colspan="2"><strong>Optional Parameters</strong></td></tr>
+<tr>
+  <td style="padding-right: 40px; width: 140px;">op</td>
+  <td>Operation type. Allowed values: <code>append</code>.</td>
+</tr>
+</table>
+
+The request payload is an object representing the attributes to append or update. The object follows
+the JSON entity representation format (described in a section above), except that `id` and `type`
+are not allowed.
+
+The entity attributes are updated with the ones in the payload. In particular, depending on
+whether `op=append` is used:
+
+* If `op=append` is not used: the entity attributes are updated (if they previously exist) or appended
+  (in they don't previously exists) with the ones in the payload.
+* If `op=append` is used (i.e. strict append semantics): all the attributes in the payload not previously
+  existing in the entity are appended. In addition to that, in the case some of the attributes in the
+  payload already exist in the entity, an error if returned.
+
```

Comment:
instead of op=append we may have options=append , but that would be part of another discussion